sonar-project.properties

```diff
@@ -3,7 +3,7 @@ sonar.organization=interstellar
 sonar.python.coverage.reportPaths=.coverage-reports/coverage.xml
 sonar.coverage.exclusions=**/tests/**
 sonar.exclusions=.coverage-reports/**,docs/**
-sonar.python.version=3.8
+sonar.python.version=3.11
```

Comment:
Why not 3.9?

Reply:
Let's use more recent version of python rather than using old ones. In fact, I would rather propose to make it 3.12

Reply:
Then we can get problems with Sonar complaining about not using latest syntax improvements or using outdated ones

Reply:
Like `Use a union type expression for this type hint.` that you have already got in the latest Sonar check

Reply:
I bumped it to 3.12, seems sonar is happy now.